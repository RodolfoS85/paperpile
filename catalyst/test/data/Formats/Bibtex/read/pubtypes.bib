@ARTICLE{article-minimal,
   author = "Leslie A. Aamport and Joe-Bob Missilany",
   title = {The Gnats and Gnus Document Preparation System},
   journal = {G-Animal's Journal},
   year = 1986,
}

@INBOOK{inbook-minimal,
   author = "Donald E. Knuth",
   title = "Fundamental Algorithms",
   publisher = "Addison-Wesley",
   year = "1973",
   chapter = "1.2",
}

@BOOK{book-minimal,
   author = "Donald E. Knuth",
   title = "Seminumerical Algorithms",
   publisher = "Addison-Wesley",
   year = "1981",
}

@INCOLLECTION{incollection-minimal,
   author = "Daniel D. Lincoll",
   title = "Semigroups of Recurrences",
   booktitle = "High Speed Computer and Algorithm Organization",
   publisher = "Academic Press",
   year = 1977,
}

@MANUAL{manual-minimal,
   title = "The Definitive Computer Manual",
}

@MASTERSTHESIS{mastersthesis-minimal,
   author = "{\'{E}}douard Masterly",
   title = "Mastering Thesis Writing",
   school = "Stanford University",
   year = 1988,
}

@MISC{misc-minimal,
   note = "This is a minimal MISC entry",
}

@INPROCEEDINGS{inproceedings-minimal,
   author = "Alfred V. Oaho and Jeffrey D. Ullman and Mihalis Yannakakis",
   title = "On Notions of Information Transfer in {VLSI} Circuits",
   booktitle = "Proc. Fifteenth Annual ACM Symposium on the Theory of Computing",
   year = 1983,
}

@PROCEEDINGS{proceedings-minimal,
   title = "Proc. Fifteenth Annual Symposium on the Theory of Computing",
   year = 1983,
}


@PHDTHESIS{phdthesis-minimal,
   author = "F. Phidias Phony-Baloney",
   title = "Fighting Fire with Fire: Festooning {F}rench Phrases",
   school = "Fanstord University",
   year = 1988,
}


@TECHREPORT{techreport-minimal,
   author = "Tom Terrific",
   title = "An {$O(n \log n / \! \log\log n)$} Sorting Algorithm",
   year = 1988,
}

@UNPUBLISHED{unpublished-minimal,
   author = "Ulrich {\"{U}}nderwood",
   title = "Lower Bounds for Wishful Research Results",
   note = "Talk at Fanstord University (this is a minimal UNPUBLISHED entry)",
}
