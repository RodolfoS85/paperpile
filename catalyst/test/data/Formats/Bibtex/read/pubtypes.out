---
pubtype: ARTICLE
---
pubtype: INBOOK
---
pubtype: BOOK
---
pubtype: INCOLLECTION
---
pubtype: MANUAL
---
pubtype: MASTERSTHESIS
---
pubtype: MISC
---
pubtype: INPROCEEDINGS
---
pubtype: PROCEEDINGS
---
pubtype: PHDTHESIS
---
pubtype: TECHREPORT
---
pubtype: UNPUBLISHED
